### ISSUE TITLE
Question: Warning: importing 'simtk.openmm' is deprecated

### ISSUE THREAD

**Issue by MichaelCato** [YoshitakaMo/localcolabfold#241]
I have finished my first run with colabfold. Everything seems to work and I get output files, including pdbs for all sequences referenced. However, at the very beginning of the run, I got this message: Warning: importing 'simtk.openmm' is deprecated.  Import 'openmm' instead.

Am I good to continue?

**Comment by YoshitakaMo** [comment]
Yes, the warning message does not harm the calculation. I can suppress the message but haven't dealt with it yet.
